app/src/main/java/org/simple/clinic/editpatient/deletepatient/DeletePatientScreen.kt

```diff
@@ -1,6 +1,7 @@
 package org.simple.clinic.editpatient.deletepatient
```

Comment:
Commit message does not make sense, can you reword it better?

Reply:
I think the commit message specifies exactly what we are doing here. We are injecting into `DeletePatientScreen` in `onAttach`. 🤔